src/rust/ide/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -413,8 +436,13 @@ impl ContainerModel {
             self.fullscreen_view.background_dom.dom().set_style_or_warn("width", "0", &self.logger);
             self.fullscreen_view.background_dom.dom().set_style_or_warn("height", "0", &self.logger);
             // self.fullscreen_view.background.shape.sprite.size.set(zero());
+
+            let quick_action_size = Vector2::new(size.x, QUICK_ACTION_BAR_HEIGHT);
+            self.action_bar.frp.set_size.emit(quick_action_size);
         }
 
+        self.action_bar.set_position_y((size.y - QUICK_ACTION_BAR_HEIGHT) / 2.0);
```

Comment:
sometimes its named "action bar" and soemetimes "quick action bar" - this sohuld be unified